.github/workflows/continuous-integration.yml

```diff
@@ -539,7 +539,7 @@ jobs:
             /github/home/.cache/Cypress
             node_modules
           key: ${{ steps.get-node-version.outputs.NODE_VERSION }}-${{ hashFiles('yarn.lock') }}-cypress
-          # restore-keys: ${{ steps.get-node-version.outputs.NODE_VERSION }}-
+          restore-keys: ${{ steps.get-node-version.outputs.NODE_VERSION }}-
```

Comment:
Should this key have `-cypress` at the end?

Reply:
We could (and I actually would be in favor like we have it in `content-build`). I would probably suggestion having it in a separate PR as this one is more of a patch for Brooks PR to unblock VFS in `vets-website`.

Do you want me to take lead on it or can you create one @ddzz ?